components/_nav-list.scss

```diff
@@ -25,8 +25,14 @@ a.tws-logo-type {
 }
 
 .nav-links {
+  text-align: center;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets